README.md

```diff
@@ -22,6 +22,10 @@ how to create directories and files on the command line of a terminal program.
 
 ## Installation
 
+** Docker users**: _The short way_
```

Comment:
I think the space between `**` and `Docker` is throwing off the formatting. I also think we can get rid of "_The short way_" and make the rest a proper short paragraph, instead of a single list item containing child list with a single item.